src/test/packages/13-find-images/helm-charts-find-images-why-expected.txt

```diff
@@ -0,0 +1,197 @@
+component: demo-helm-charts
+chart: podinfo-oci
+resource: podinfo-oci-service-test-wlxqz
```

Comment:
Will need to make sure each expected txt file does not have a resource name since the implementation doesn't output this anymore

so should just be:

```
resource:

// YAML object
```

Reply:
I have updated the test also I had to make the names of the pods into regular expressions as the pod names are changing everytime